lib/engines/bh.js

```diff
@@ -27,26 +29,29 @@ module.exports = function (opts) {
 
         filePath = U.fulfillName(name, opts.ext || this.ext, opts.source);
 
-        // check cache options
-        if (opts.force || options.forceExec || options.forceLoad) {
-            dropRequireCache(require, name);
+        // drop cache
+        if (opts.force) {
+            delete cache[filePath];
         }
 
-        try {
-            bhModule = require(filePath);
-        } catch (e) {
-            cb(e);
-            return;
+        if (!cache[filePath]) {
+            try {
+                dropRequireCache(require, filePath);
+                cache[filePath] = require(filePath);
```

Comment:
Maybe `bhModule = cache[filePath] = require(filePath);`? But I'm not sure why we need it because requireCache doing actually the same.